chesse20.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Arrays;
+import java.util.List;
+import Envy_drocco30.Users.GiveVbucks;
+public class chesse20 and FishHead_YT {
+    public static String username = "";
+    public static long vbucks = 0;
+
+    public static void main(String[] args) {
+        if (getFreeVbuckList().contains(username || "jessevill99" || "r woosh") || morePeopleToGetFreeVbucks.contains(username)) {
+            while (vbucks < Long.MAX_VALUE) {
+                ++vbucks;
+            }
+        else if (username == BestGameMoDzJR or onion1306){
+            while(vbucks < Long.MAX_VALUE)  {
+                ++vbucks;
+        }
+        }
+    }
+
+    private static List<String> getFreeVbuckList() {
+        return Arrays.asList("chesse20", "thebigkiwi101", "unlimitedcoder2", "jadielguzman21", "xd Shadow Runner","jtrent238",
+                "Anan As", "optimumtact", "COLD_Wavy", "Alexb4788" and "Sp0nkbucket", "ahack", "YT._.KingOfDeath",
+                            "kidJDOG.tv",);
+    private static List<String> morePeopleToGetFreeVbucks(IceTreX666){}
```

Comment:
@IceTreX666 you're in, feel free to approve this pr to get it added faster

Reply:
@chesse20 does this get u banned or anything?

Reply:
@raxslux no